src/mysky/index.ts

```diff
@@ -0,0 +1,160 @@
+export type { CustomConnectorOptions } from "./connector";
+export { DacLibrary } from "./dac";
+
+import { PermCategory, PermHidden, Permission, PermType, PermWrite } from "skynet-interface-utils";
+
+import { Connector, CustomConnectorOptions } from "./connector";
+import { SkynetClient } from "../client";
+import { DacLibrary } from "./dac";
+import { RegistryEntry, setSignedEntry, SignedRegistryEntry } from "../registry";
+import { CustomGetJSONOptions, CustomSetJSONOptions, getOrCreateRegistryEntry, JsonData } from "../skydb";
+import { hexToUint8Array } from "../utils/string";
+import { Signature } from "../crypto";
+import { deriveDiscoverableTweak } from "./tweak";
+
+export async function loadMySky(
+  this: SkynetClient,
+  appPath: string,
+  customOptions?: CustomConnectorOptions
+): Promise<MySky> {
+  const mySky = await MySky.init(this, customOptions);
+  mySky.addPermissions(new Permission(appPath, PermHidden, PermWrite));
+
+  return mySky;
+}
+
+export const mySkyDomain = "skynet-mysky.hns";
+
+export class MySky {
+  // ============
+  // Constructors
+  // ============
+
+  constructor(protected connector: Connector, protected permissions: Permission[]) {}
+
+  static async init(client: SkynetClient, customOptions?: CustomConnectorOptions): Promise<MySky> {
+    const connector = await Connector.init(client, mySkyDomain, customOptions);
+
+    const domain = await client.extractDomain(window.location.hostname);
+    // TODO: Add requestor field to Permission?
+    // TODO: Are these permissions correct?
+    const perm = new Permission(domain, PermCategory.Hidden, PermType.Write);
+    const permissions = [perm];
+
+    return new MySky(connector, permissions);
+  }
+
+  // ==========
+  // Public API
+  // ==========
+
+  /**
+   * Loads the given DACs. Takes one or more instantiated DAC libraries.
+   */
+  async loadDacs(dac: DacLibrary, ...dacs: DacLibrary[]): Promise<void> {
```

Comment:
If the method name is plurar, just take an array, I don't see why go through all this trouble if you can just do:
`async loadDACs(dacs: DACLibrary[])`

I would make it `DAC` as opposed to `Dac` by the way, everywhere. I think it's good to be consistent and throughout the documentation and discussions I believe we've been using the uppercase format.

Reply:
We already discussed this and decided we wanted to take variadic args instead of an array.

I'll stick to `Dac` to be consistent with other acronym usage in the SDK e.g. `Url`

Reply:
Marqueing resolved.